charts/nextcloud/README.md

```diff
@@ -204,6 +205,38 @@ The following table lists the configurable parameters of the nextcloud chart and
 | `podAnnotations`                                            | Annotations to be added at 'pod' level                                                              | not set                    |
 | `dnsConfig`                                                 | Custom dnsConfig for nextcloud containers                                                           | `{}`                       |
 
+### Ingress
+#### Ingress Sticky-Sessions
+
+For loadbalance over multiple Pods, it is useful to configure sticky session.
```

Comment:
Maybe some further explanation when this is necessary would be good, but we can add that later (it should already be documented in the docs specific to the apps that need it).